tests/test_framework/utils.py

```diff
@@ -17,18 +18,18 @@
 
 from ephemeral_port_reserve import reserve
 from test_framework import serializations
-from typing import Optional
+from typing import Optional, Tuple
 
 
-TIMEOUT = int(os.getenv("TIMEOUT", 60))
-TEST_DEBUG = os.getenv("TEST_DEBUG", "0") == "1"
+TIMEOUT = int(os.getenv("TIMEOUT", 100))
+TEST_DEBUG = os.getenv("TEST_DEBUG", "1") == "1"
 EXECUTOR_WORKERS = int(os.getenv("EXECUTOR_WORKERS", 20))
 POSTGRES_USER = os.getenv("POSTGRES_USER", "")
 POSTGRES_PASS = os.getenv("POSTGRES_PASS", "")
 POSTGRES_HOST = os.getenv("POSTGRES_HOST", "localhost")
 POSTGRES_IS_SETUP = POSTGRES_USER and POSTGRES_PASS and POSTGRES_HOST
-VERBOSE = os.getenv("VERBOSE", "0") == "1"
-LOG_LEVEL = os.getenv("LOG_LEVEL", "trace")
+VERBOSE = os.getenv("VERBOSE", "1") == "1"
+LOG_LEVEL = os.getenv("LOG_LEVEL", "debug")
```

Comment:
Although i  think it makes sense, this change is unrelated and should *at least* be in another commit

Reply:
This wasn't addressed?